core/src/filter/ast.rs

```diff
@@ -837,6 +838,7 @@ impl fmt::Display for Value {
             Value::Ipv4(net) => write!(f, "{}", net),
             Value::Ipv6(net) => write!(f, "{}", net),
             Value::Text(val) => write!(f, "{}", val),
+            Value::Byte(val) => write!(f, "{:?}", val),
```

Comment:
Can you format this to print in a consistent format with the input? With the || and in hex.

For hex, might need an `iter().for_each` to format each byte individually in the Vec<u8> -- I don't remember if there's an easier way.